mycli/packages/parseutils.py

```diff
@@ -64,6 +64,9 @@ def last_word(text, include='alphanum_underscore'):
 # This code is borrowed from sqlparse example script.
 # <url>
 def is_subselect(parsed):
+    if isinstance(parsed, sqlparse.sql.Token):
+        if not parsed.is_group:
+            return False
     if not parsed.is_group():
```

Comment:
@meeuw Thanks for helping out with mycli! Is this call to `is_group()` still supposed to be in here?